app/Helpers/Fixometer.php

```diff
@@ -140,21 +141,15 @@ public static function userHasEditPartyPermission($partyId, $userId = null)
         }
 
         if (self::hasRole($user, 'NetworkCoordinator')) {
-            $group = $party->theGroup;
             foreach ($group->networks as $network) {
                 if ($network->coordinators->contains($user)) {
                     return true;
                 }
             }
         }
 
-        if (self::hasRole($user, 'Host')) {
-            $group_id_of_event = Party::where('idevents', $partyId)->value('group');
-            if (self::userIsHostOfGroup($group_id_of_event, $userId)) {
-                return true;
-            } elseif (empty(DB::table('events_users')->where('event', $partyId)->where('user', $user->id)->first())) {
-                return false;
-            }
+        if (self::userIsHostOfGroup($group->idgroups, $userId)) {
```

Comment:
That removed logic looks a bit odd - it looks like it was once attempting to check whether the user has any kind of association to the event?  Makes me wonder if elsewhere is using it to check whether someone can edit the items at an event.

But then essentially it looks like it was a no-op because it was returning false.

So as I'm guessing that the tests are all still passing, then this change looks good :+1:

Reply:
Ah right, so it looks like it was trying to ensure that a 'Host' who was not a host of that group not able to edit the party.  Which is exactly what userIsHostOfGroup does.  So all good!